src/components/CategoryRadar.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { FC } from "react";
 import * as d3 from "d3";
 import styled from "styled-components/macro";
-import { CategoryRadarTooltip } from "../components/StyledTooltip";
+import { CategoryRadarTooltip } from "./StyledTooltip";
```

Comment:
As the StyledTooltip is placed in same component folder as CategoryRadar, hence updated the path

Reply:
This is no longer valid, changes were overridden by master branch merge